app/presenters/hyrax/asset_presenter.rb

```diff
@@ -3,6 +3,8 @@
 module Hyrax
   class AssetPresenter < Hyrax::WorkShowPresenter
 
+    attr_reader :batch, :batch_url, :batch_ingest_label, :batch_ingest_date
```

Comment:
@jasoncorum these aren't needed if we have explicitly created accessor methods below.